exercises/change/change.js

```diff
@@ -0,0 +1,14 @@
+//
+// This is only a SKELETON file for the 'Allergies' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+export class Candidate {
+  constructor() {
+    throw new Error("Remove this statement and implement this function");
+  }
+
+  addCoin() {
```

Comment:
The function expected is actually called `calculate` and receives _two_ arguments.